models/workitem_repository_blackbox_test.go

```diff
@@ -0,0 +1,80 @@
+package models_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/criteria"
+	"github.com/almighty/almighty-core/models"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateLoadDeleteListWI(t *testing.T) {
+	doWithTransaction(t, func(ts *models.GormTransactionSupport) {
```

Comment:
The idea that you just have to look at a test to know what is broken is a red herring in most cases. You'll have to reproduce and debug the code anyway. The target code of this test  is colocated and highly coupled. I don't want to copy/paste the setup code x times just to satisfy an abstract notion of niceness. Our guidelines should favor easy & fast writing of tests.
What I'm saying is splitting this test method into 5 different ones will not increase coverage and it won't allow us to debug failures faster. At least I'm convinced of that.